src/Microsoft.AspNetCore.Blazor.Razor.Extensions/BindLoweringPass.cs

```diff
@@ -1,17 +1,17 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System;
-using System.Collections.Generic;
 using System.Linq;
 using Microsoft.AspNetCore.Razor.Language;
-using Microsoft.AspNetCore.Razor.Language.Extensions;
 using Microsoft.AspNetCore.Razor.Language.Intermediate;
 
 namespace Microsoft.AspNetCore.Blazor.Razor
 {
     internal class BindLoweringPass : IntermediateNodePassBase, IRazorOptimizationPass
     {
+        // Run after event handler pass
+        public override int Order => base.Order + 50;
```

Comment:
No need to change this now, but longer term I wonder if it would be helpful to move all these ordering values out into a central location somewhere. Then we'd be able to visualize all the steps and the order they run in.

Reply:
Or another idea, we could just list them in order where they are registered